client/extensions/zoninator/components/settings/zone/post-card.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+
+class PostCard extends Component {
+
+	handleRemove = () => {
+		this.props.remove( this.props.cardIdx );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+
+	render() {
+		const {
+			translate,
+			post: {
+				ID,
+				URL,
+				site_ID,
+				title,
+				slug,
+			}
+		} = this.props;
+
+		const postCardClass = 'zoninator__zone__list-item';
+
+		return (
+			<SectionHeader key={ slug } label={ title } className={ postCardClass }>
+				<Button
+					compact
+					onMouseDown={ this.handleMouseDown }
+					href={ URL }
+					target="_blank"
```

Comment:
Opening this in a new tab is likely fine, since the post is outside Calypso.

Unless :grin: you use one of our preview components. Maybe a team Link member can advise. cc @mcsf @marekhrabe

Reply:
(Meaning there might be an action that you can call `onClick` here that fires up a preview of that URL.)

Reply:
I see that the posts open in a new tab when clicking on the `View` button in the `Blog posts` section of Calypso, so I'm going to stick with that for now.  
But I will note this down for possible future iterations.